src/runtime/components/SelectMenu.vue

```diff
@@ -204,33 +203,54 @@ function displayValue(value: T | T[]): string {
   return item && (typeof item === 'object' ? get(item, props.labelKey as string) : item)
 }
 
+function filterFunction(
```

Comment:
Note - we can get rid of this filterFunction; we can make use of `useFilter` from `reka-ui`. I just wonder about custom filter option when the guide for it is completed 🤔

Reply:
Also, to access the filtered options, we would need to use ComboboxRootContext (https://reka-ui.com/docs/guides/inject-context) so the SFC would need to be split

Reply:
Yeah we can't make our own filter with `escapeRegExp` because it won't necessarily match the filter of Reka UI.

Reply:
Can you advise on if/how we should split the SFC to access the Combobox context? (as it should be child components)

Reply:
@zernonia Would you have some hint about this? What would be the best way to access the filtered items?

Reply:
I see https://github.com/unovue/radix-vue/commit/d54a4481718d305c8405c3ec4dc4dc54a4bdfabc exposes `filterState` now to `defineExpose` now. Will progress with this 👍

Reply:
Docs for custom filtering have been updated in Reka - https://reka-ui.com/docs/components/combobox#custom-filtering

Reply:
@benjamincanac I think based on the above link, we can re-add `filterFunction` that allows `filter` prop as array of strings.

Reka will do search only on the display value, while our function was a bit more powerful.
https://github.com/unovue/radix-vue/blob/8f406899ce3f848f77459d8948eb83e3295d8251/packages/core/src/Combobox/ComboboxItem.vue#L65